src/components/Icon/assets/generic/keyword.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
remove width and height attributes as they alter the final size of the icon

Reply:
resolved at https://github.com/Orfium/orfium-ictinus/pull/823/commits/a3879a6f77774c1d9d1257360457ec6bebbb5f79